src/Discord.Net.SlashCommands/Attributes/ChoiceAttribute.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+
+namespace Discord.SlashCommands
+{
+    /// <summary>
+    /// Add a pre-determined argument value to a command parameter
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true, Inherited = true)]
+    public class ChoiceAttribute : Attribute
```

Comment:
Attributes dont accept complex object types or 2+ dimensional arrays. We can make the attribute take two arrays, one containing the names and one containing the values but this makes it really hard to read and it is much more messy than the current solution. Also i think people will hardly use this attribute since enums are automatically resggitered as options with choices by the library. I think using enums are the way to go.